--- a/setup.py
+++ b/setup.py
@@ -49,8 +49,6 @@ setup(name='ViTables',
           'Programming Language :: Python',
           'Topic :: Scientific/Engineering'
       ],
-      install_requires=['qtpy (>=1.2.1)', 'PyQt5 (>=5.5.1)', 'numpy (>=1.4.1)',
-                'numexpr (>=2.0)', 'tables (>=3.0)'],
       entry_points={
           'gui_scripts': ['vitables = vitables.start:gui'],
           'vitables.plugins':
